web-frontend/src/pages/TransferCoins/SendCoins/Send.tsx

```diff
@@ -13,16 +10,18 @@ import {
 import { SendForm } from './SendForm';
 import { SendConfirmation } from './SendConfirmation';
 
-function Send() {
-  const { nickname } = useParams();
-  const { data: accounts = [] } = useResource<AccountObject[]>('accounts');
-  let account = accounts.find((account) => account.nickname === nickname);
+function Send({ ...props }) {
+  const { account } = props;
+  const nickname = account.nickname ?? '';
+  const queryParams = new URLSearchParams(location.search);
+  const presetTo = queryParams.get('to') ?? '';
+  const presetAmount = queryParams.get('amount') ?? '';
```

Comment:
You don't need that!

You can use the hook useParam

import { Routes, Route, useParams } from 'react-router-dom';
```ts
function ProfilePage() {
  // Get the userId param from the URL.
  let { to, amount, ... } = useParams();
  // ...
}
```

Reply:
it didn't work, i think useParams id for `baseURL/param1/***` and I am using `url?param1=value`